### PR TITLE
Add environment cache docs

### PR DESCRIPTION
**Pull request by matthewelwell**
## Changes

Adds documentation for configuring the environment cache.

### CODE REVIEW

File: docs/deployment/hosting/locally-api.md [hunk 1]
```diff
@@ -454,6 +455,18 @@ GET_IDENTITIES_ENDPOINT_CACHE_BACKEND: django.core.cache.backends.memcached.PyMe
...
+
+| Environment Variable                    | Description                                                                                                                    | Example value                                          | Default                                       |
+| --------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------ | --------------------------------------------- |
+| <code>ENVIRONMENT_CACHE_SECONDS</code>  | Number of seconds to cache the environment for                                                                                 | `60`                                                   | `86400` ( = 24h)                              |
+| <code>ENVIRONMENT_CACHE_BACKEND</code>  | Python path to the django cache backend chosen. See documentation [here](https://docs.djangoproject.com/en/3.2/topics/cache/). | `django.core.cache.backends.memcached.PyMemcacheCache` | `django.core.cache.backends.dummy.DummyCache` |
```
Comment (dabeeeenster) [inline docs/deployment/hosting/locally-api.md:467]:
```suggestion
| `ENVIRONMENT_CACHE_BACKEND`  | Python path to the django cache backend chosen. See documentation [here](https://docs.djangoproject.com/en/3.2/topics/cache/). | `django.core.cache.backends.memcached.PyMemcacheCache` | `django.core.cache.backends.dummy.DummyCache` |
```

File: docs/deployment/hosting/locally-api.md [hunk 2]
```diff
@@ -454,6 +455,18 @@ GET_IDENTITIES_ENDPOINT_CACHE_BACKEND: django.core.cache.backends.memcached.PyMe
...
+| <code>ENVIRONMENT_CACHE_LOCATION</code> | The location for the cache. See documentation [here](https://docs.djangoproject.com/en/3.2/topics/cache/).                     | `127.0.0.1:11211`                                      | `environment-objects`                         |
```
Comment (dabeeeenster) [inline docs/deployment/hosting/locally-api.md:468]:
```suggestion
| `ENVIRONMENT_CACHE_LOCATION` | The location for the cache. See documentation [here](https://docs.djangoproject.com/en/3.2/topics/cache/).                     | `127.0.0.1:11211`                                      | `environment-objects`                         |
```

**Review by dabeeeenster — COMMENTED** [review]
Not sure where <code> came from?!

**Comment by matthewelwell** [comment]
I have a feeling it's required in tables... Let me check with `

**Pull request merged**